tests/validation/tests/v3_api/test_aks_v2_hosted_cluster.py

```diff
@@ -121,15 +121,4 @@ def create_and_validate_aks_cluster(cluster_config, imported=False):
                                check_intermediate_state=intermediate_state,
                                skipIngresscheck=True,
                                timeout=DEFAULT_TIMEOUT_AKS)
-    return client, cluster
-
-
-@pytest.fixture(scope='module', autouse="False")
```

Comment:
we're going to have to delete the cleanup method?

Reply:
Yeah, otherwise the certification tests never run and the cluster keeps deleting itself after it is provisioned, regardless if autouse is defined or not.